survey_calculator/models/survey_calculator_result.py

```diff
@@ -0,0 +1,169 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Odoo, Open Source Management Solution
+#    This module copyright (C) 2015 Savoir-faire Linux
+#    (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import api, models, fields
+from openerp.tools import ustr
+from openerp.tools.safe_eval import safe_eval
+
+
+class SurveyCalculatorResult(models.Model):
+    _name = 'survey.calculator.result'
+    _description = "Survey Calculator Result"
+
+    user_input_id = fields.Many2one(
+        'survey.user_input',
+        'User Input',
+        required=True,
+        help='The user input (answers) on which the calculation will be '
+             'applied.',
+        domain='[("state", "=", "done")]'
+    )
+    partner_id = fields.Many2one(
+        'res.partner',
+        'Partner',
+        related='user_input_id.partner_id',
+        readonly=True,
+        store=True,
+    )
+    survey_id = fields.Many2one(
+        'survey.survey',
+        'Survey',
+        related='user_input_id.survey_id',
+        readonly=True,
+        help='Survey used to compute results',
```

Comment:
Those periods are here in order to simplify the git diff. This way, when you add a new argument, only 1 line will be on the git diff, not 2.